opentelemetry-sdk/src/opentelemetry/sdk/_metrics/aggregation.py

```diff
@@ -16,82 +16,138 @@
 from collections import OrderedDict
 from logging import getLogger
 from math import inf
+from threading import Lock
+from typing import Generic, Optional, Sequence, TypeVar
 
-from opentelemetry._metrics.instrument import _Monotonic
+from opentelemetry.sdk._metrics.measurement import Measurement
+from opentelemetry.sdk._metrics.point import Gauge, Histogram, PointT, Sum
 from opentelemetry.util._time import _time_ns
 
+# FIXME this is being copied directly from
```

Comment:
I don't think there's anything to fix, there's no workaround really. Can we just make a Python enum which is easy to use? It should be publicly exposed so probably in `point.py`

Reply:
All right, added an enum. Since it is an aggregation temporality enum, I think it belongs in `aggregation.py` instead :shrug: